### PR TITLE
fixing shadow declaration and dockerfile

### PR DESCRIPTION
**Pull request by dionisislolas**
### What

Fixing shadow declaration and dockerfile

### How to review

Check changes are ok.

### Who can review

Anyone

### CODE REVIEW

File: mapper/mapper_test.go [hunk 1]
```diff
@@ -77,6 +77,15 @@ var availableItems = []model.Topics{
 		ID:    "4567",
 	},
 }
+var cfg *config.Config
+
+func init() {
+	var err error
+	cfg, err = config.Get()
+	if err != nil {
+		panic("failed to get config")
```
Comment (gedge) [inline mapper/mapper_test.go:86]: quick thought:
```suggestion
		panic("failed to get config" + err.Error())
```
needs the error message, too

File: Dockerfile.local [hunk 2]
```diff
@@ -4,6 +4,7 @@ ENV GOCACHE=/go/.go/cache GOPATH=/go/.go/path TZ=Europe/London PATH=$PATH:$GOPAT
 
 # Install github.com/cespare/reflex
 RUN GOBIN=/bin go install github.com/cespare/reflex@v0.3.1
+RUN GOBIN=/bin go install github.com/kevinburke/go-bindata/v4/...@latest
```
Comment (redhug1) [inline Dockerfile.local:7]: You should not be using @latest, because this could pull in bad code.
This needs to be pinned to a version number.

**Review by redhug1 — CHANGES_REQUESTED** [review]
I've only looked at this one issue, others need to check the rest.

File: Dockerfile.local (hunk 2 above)
Comment (dionisislolas) [reply Dockerfile.local]: This is now addressed.

**Review by redhug1 — APPROVED** [review]
LGTM

**Pull request merged**